wayfire.ini.default

```diff
@@ -1,291 +0,0 @@
-vwidth = 3
-# number of vertical workspaces
-vheight = 2
-
-# Send close request to the currently focused view
-close_top_view = <super> KEY_Q | <alt> KEY_FN_F4
-
-# start Xwayland, which provides support for running X applications
-xwayland = 1
-
-# background color of workspaces
-# visible when nothing is drawing the background
-background_color = 0 0 0 1
-
-# Preferred decoration mode: server | client
-preferred_decoration_mode = client
-
-# apps that should run on startup. any backgrounds/panels belong here
-# by default, wayfire tries to run the clients from
-# https://github.com/WayfireWM/wf-shell
-[autostart]
-background = wf-background
-panel = wf-panel
-
-# run a certain application when the user presses a keybinding
-# useful to setup a screenshot, brightness control, sound control bindings
-# or any other application that you want
-
-[command]
-binding_1 = <super> KEY_D
-command_1 = dmenu_run
-binding_2 = <super> KEY_ENTER
-command_2 = tilix
-binding_3 = KEY_VOLUMEUP
-command_3 = wf-sound-control i 5
-binding_4 = KEY_VOLUMEDOWN
-command_4 = wf-sound-control d 5
-binding_5 = KEY_MUTE
-command_5 = amixer -q sset Master toggle
-
-binding_lock = <super> KEY_L
-command_lock = swaylock
-# Repeatable bindings allow the user to repeat the given command by holding the
-# key or button with which the binding was activated.
-repeatable_binding_echo = <ctrl> <alt> <super> <shift> KEY_E
-command_echo = echo "Test"
-
-# input options
-[input]
-# -1..1, where 0 is default
-mouse_cursor_speed = 0
-touchpad_cursor_speed = 0
-# 0..* (multipliers)
-mouse_scroll_speed = 1
-touchpad_scroll_speed = 1
-
-mouse_accel_profile = default    # none | adaptive | flat
-touchpad_accel_profile = default # none | adaptive | flat
-
-natural_scroll = 1
-tap_to_click = 1
-click_method = default  # none | button-areas | clickfinger
-scroll_method = default # none | two-finger | edge | on-button-down
-disable_while_typing = 1
-disable_touchpad_while_mouse = 0
-
-cursor_size = 24
-cursor_theme = default
-
-#xkb_layout = us,bg
-#xkb_option = grp:win_space_toggle,compose:ralt
-#xkb_variant = ,phonetic
-
-kb_repeat_rate = 40
-kb_repeat_delay = 400
-
-# cancel modifier actions (like <super> for expo) when held for this long, 0 to never cancel
-modifier_binding_timeout = 0
-
-# output configuration
-# overlapping outputs are not supported
-[eDP-1]
-# position of top-left corner in global compositor space
-layout = 0@0
-#add a custom mode, can be used in the mode option
-custom_mode_1 = 74.50  1280 1344 1472 1664  720 723 728 748 -hsync +vsync
-#set resolution
-#mode = 1280x720@60.000
-# set output scale
-scale = 1.00
-#set rotation
-transform = normal
-
-# change window alpha with modifier + scroll
-[alpha]
-min_value = 0.01
-modifier = <alt>
-
-# fisheye effect
-[fisheye]
-radius = 450
-zoom = 8.910000
-toggle = <super> KEY_F
-
-# wobbly windows
-[wobbly]
-spring_k = 1
-friction = 1
-grid_resolution = 7
-
-# bind a certain input device to an output
```

Comment:
This is also useful to know, but maybe it belongs to the wiki and not to the config file ...